.github/workflows/charts-release.yaml

```diff
@@ -86,20 +86,21 @@ jobs:
         run: |
           #!/bin/bash
           CHARTS=(${{ steps.collect-changes.outputs.modifiedChartsAfterTag }})
+          mkdir -p website/static/img/chart-icons || echo "chart-icons path already exists, continuing..."
           for i in "${CHARTS[@]}"
           do
               IFS='/' read -r -a chart_parts <<< "$i"
               if [ -f "charts/${chart_parts[0]}"/"${chart_parts[1]}/Chart.yaml" ]; then
                 train=${chart_parts[0]}
                 chart=${chart_parts[1]}
                 echo "copying docs to website for ${chart}"
-                mkdir -p docs/apps/${train}/${chart} || echo "app path already exists, continuing..."
-                yes | cp -rf charts/${train}/${chart}/README.md pub/website/apps/${train}/${chart}/index.md 2>/dev/null || :
-                yes | cp -rf charts/${train}/${chart}/CHANGELOG.md pub/website/apps/${train}/${chart}/CHANGELOG.md 2>/dev/null || :
-                yes | cp -rf charts/${train}/${chart}/security.md pub/website/apps/${train}/${chart}/security.md 2>/dev/null || :
-                rm pub/website/apps/${train}/${chart}/LICENSE.md 2>/dev/null || :
-                yes | cp -rf charts/${train}/${chart}/LICENSE pub/website/apps/${train}/${chart}/LICENSE.md 2>/dev/null || :
-                sed -i '1s/^/# License<br>\n\n/' pub/website/apps/${train}/${chart}/LICENSE.md 2>/dev/null || :
+                mkdir -p website/docs/charts/${train}/${chart} || echo "chart path already exists, continuing..."
+                yes | cp -rf charts/${train}/${chart}/README.md website/docs/charts/${train}/${chart}/index.md 2>/dev/null || :
+                yes | cp -rf charts/${train}/${chart}/CHANGELOG.md website/docs/charts/${train}/${chart}/CHANGELOG.md 2>/dev/null || :
+                yes | cp -rf charts/${train}/${chart}/security.md website/docs/charts/${train}/${chart}/security.md 2>/dev/null || :
```

Comment:
we're not going to build security here anymore due to it taking a **LONG** time to render...

```suggestion
```